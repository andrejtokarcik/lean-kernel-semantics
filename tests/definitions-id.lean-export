1 #NS 0 id
2 #NS 0 l
3 #NS 0 A
1 #UP 2
0 #ES 1
4 #NS 0 a
1 #EV 0
2 #EV 1
3 #EP #BD 4 1 2
4 #EP #BI 3 0 3
5 #EL #BD 4 1 1
6 #EL #BI 3 0 5
#DEF 1 4 6 2
