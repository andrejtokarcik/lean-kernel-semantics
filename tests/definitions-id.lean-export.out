<T>
  <k>
    .
  </k>
  <names>
    0 |-> *empty*
    1 |-> id
    2 |-> l
    3 |-> A
    4 |-> a
  </names>
  <universes>
    0 |-> 0
    1 |-> l
  </universes>
  <expressions>
    0 |-> l
    1 |-> #0
    2 |-> #1
    3 |-> Π (a : #0) → #1
    4 |-> Π {A : l} → Π (a : #0) → #1
    5 |-> λ (a : #0) → #0
    6 |-> λ {A : l} → λ (a : #0) → #0
  </expressions>
  <def-types>
    id |-> Π {A : l} → Π (a : #0) → #1
  </def-types>
  <def-values>
    id |-> λ {A : l} → λ (a : #0) → #0
  </def-values>
  <exit-code>
    0
  </exit-code>
</T>
